packages/components/src/navigation/menu.js

```diff
@@ -0,0 +1,5 @@
+const NavigationMenu = ( { children } ) => {
+	return <ul className="components-navigation__menu">{ children }</ul>;
+};
```

Comment:
Can the `<Navigation />` component absorb this `ul`?  We can still have the separation of different sections by filtering the `<NavigationMenuItem />`'s according to primary or secondary in the implementation.

Reply:
I don't think we can.  If `Navigation` absorbs the `ul` then we will no longer be able to include other non-`li` elements under here.

Reply:
Right, good point.